src/script/media/MediaDevicesHandler.ts

```diff
@@ -185,30 +185,25 @@ export class MediaDevicesHandler {
         this._removeAllDevices();
 
         if (mediaDevices) {
-          const audioInputDevices: MediaDeviceInfo[] = [];
-          const audioOutputDevices: MediaDeviceInfo[] = [];
-          const videoInputDevices: MediaDeviceInfo[] = [];
-
-          mediaDevices.forEach(mediaDevice => {
-            switch (mediaDevice.kind) {
-              case MediaDeviceType.AUDIO_INPUT: {
-                audioInputDevices.push(mediaDevice);
-                break;
-              }
-
-              case MediaDeviceType.AUDIO_OUTPUT: {
-                audioOutputDevices.push(mediaDevice);
-                break;
-              }
-
-              case MediaDeviceType.VIDEO_INPUT: {
-                videoInputDevices.push(mediaDevice);
-                break;
-              }
+          const audioOutputDevices: MediaDeviceInfo[] = mediaDevices.filter(
+            device => device.kind === MediaDeviceType.AUDIO_OUTPUT,
+          );
+          const videoInputDevices: MediaDeviceInfo[] = mediaDevices.filter(
+            device => device.kind === MediaDeviceType.VIDEO_INPUT,
+          );
+          const microphones = mediaDevices.filter(device => device.kind === MediaDeviceType.AUDIO_INPUT);
+          /**
```

Comment:
It's not JSDoc, so let's remove the second `*`.

```suggestion
          /*
```